src/platform/user/profile/vap-svc/tests/containers/AddressValidationModal.unit.spec.jsx

```diff
@@ -417,14 +417,14 @@ describe('<AddressValidationModal/>', () => {
 
     expect(
       component
-        .find('h3')
+        .find('h1')
         .at(0)
         .text(),
     ).to.equal('Edit mailing address');
     expect(
       component
         .find('h3')
-        .at(1)
+        .at(0)
```

Comment:
This is happening because of [`component-library` version `3.11.4`](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v3.11.3):

> Change modal header to h1